sw.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+  <head>
+    <title>installing service worker</title>
+    <script type="text/javascript">
+      var swsource = "{{ site.url }}/sw.js";
+      if("serviceWorker" in navigator) {
+          navigator.serviceWorker.register(swsource).then(function(reg){
+              console.log('ServiceWorker scope: ', reg.scope);
+          }).catch(function(err) {
+              console.log('ServiceWorker registration failed: ', err);
+          });
+      };
+    </script>
```

Comment:
Loads the actual service worker, normally `script` tags are forbidden with AMP sites.